src/Maintainable.java

```diff
@@ -0,0 +1,3 @@
+public interface Maintainable {
+    void check(Machine a);
```

Comment:
Этот интерфейс реализует транспортные средства, передавать его в метод не нужно
```suggestion
    void check();
```